runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -826,7 +826,14 @@ func s:InstallCommands()
   command Winbar call s:InstallWinbar()
 
   if !exists('g:termdebug_map_K') || g:termdebug_map_K
-    let s:k_map_saved = maparg('K', 'n', 0, 1)
+    " let s:k_map_saved = maparg('K', 'n', 0, 1)
+    let s:k_map_saved = {}
+    for map in nvim_get_keymap('n')
+      if map.lhs ==# 'K'
+        let s:k_map_saved = map
+        break
+      endif
+    endfor
```

Comment:
No need to change this, `maparg` works perfectly fine in Neovim.

Reply:
The thing is that, as we were discussing, `maparg` returns `<SID>` and `nvim_get_keymap` does the expansion to `<SNR>X_`.

Reply:
I just want to make sure that's actually a problem -- does `nvim_set_keymap` _break_ with `<SID> `?

Reply:
Yes, it tries to link it to the `termdebug.vim` script instead of where it was actually defined previously.